front/app/components/SignUpIn/SignUp/Consent.tsx

```diff
@@ -101,15 +100,19 @@ const Consent = memo<Props & InjectedIntlProps>(
               </ConsentText>
             }
           />
-          <Error text={tacError ? formatMessage(messages.tacError) : null} />
+          <Error
```

Comment:
Could be cleaner to render the whole `Error` component conditionally, not just the text?

Reply:
This component can definitely use some love, but I'm not going to touch it now. :)